### DISCUSSION TITLE
Formatting bold makes Junicode bold-italic

### DISCUSSION THREAD

**Discussion by dfballes** [psb1558/Junicode-font#104] [Q&A]
I downloaded and installed the newest release of Junicode, but now have a problem using it in Word 2013 and LibreOffice Writer on Win10: Every time I format text as **bold**, it appears as _**bold italic**_, without any possibility (at least for me) to change that behaviour. Switching _italic_ on and off does not change anything.
What am I doing wrong?

**Comment by psb1558** [comment]
Okay, I broke one thing while fixing another. At least I understand this one (I think) and will fix today or tomorrow. I'm moving this to issues.
